test/slurm_sync.py

```diff
@@ -207,7 +208,8 @@ def test_slurm_user_accounts(self):
             SlurmUser(User='user5', Def_Acct='vo2', Admin='None', Cluster='banette', Account='vo2', Partition='', Share='1', MaxJobs='', MaxNodes='', MaxCPUs='', MaxSubmit='', MaxWall='', MaxCPUMins='', QOS='normal', Def_QOS=''),
         ]
 
-        (job_cancel_commands, commands, remove_user_commands) = slurm_user_accounts(vo_members, active_accounts, slurm_user_info, ["banette"])
+        with patch("vsc.administration.slurm.sacctmgr.get_slurm_sacct_active_jobs_for_user", return_value="testing"):
```

Comment:
you need to patch the code where it is used, not where it is defined. so probably 
```
with patch('vsc.administration.slurm.sync.get_slurm_sacct_active_jobs_for_user', ...)
```

Reply:
that is confusing. I thought the context manager defines where it is used, so it would patch there. I don't understand the logic, because `vsc.administration.slurm.sync ` does not have `get_slurm_sacct_active_jobs_for_user`. but you are right. it works now.